code/game/objects/items/devices/scanners.dm

```diff
@@ -412,6 +416,10 @@ GENE SCANNER
 	to_chat(user, jointext(render_list, ""), trailing_newline = FALSE) // we handled the last <br> so we don't need handholding
 
 /proc/chemscan(mob/living/user, mob/living/M)
+	if(user.is_blind())
+		to_chat(user, "<span class='warning'>You realize that your scanner has no accessibility support for the blind!</span>")
+		return
```

Comment:
This is a good change, but is it related?
Could you atomize these fixes in their own PRs?